### PR TITLE
Update apk-generator.md

### PR DESCRIPTION
**Pull request by utkarshnath**
Added delivery options #1433 

@the-dagger plz review

### CODE REVIEW

**Comment by the-dagger** [comment]
@utkarshnath
LGTM, might want to add some screenshots and related code snippets too though.

**Comment by utkarshnath** [comment]
@the-dagger 
Added code snippets.
I think the b part of #1433 ie Generating the app would have screenshots, in this part what specific screenshots do you want

File: docs/apk-generator.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
+After generation of the app, we have two option for delivery of the apk
+
+ 1. ### Email
+  For email we use Marrow Mailer. We have a class Notification for sending emails at 
+ `open-event-android/apk-generator/v2/app/utils/notification.py` and functions in this class is called in `apk-generator/v2/app/generator/generator.py`
+  
+notification.py has two functions 
+- send  (under development)
+- send_mail_via_smtp_  (for sendind mails)
```
Comment (the-dagger) [inline docs/apk-generator.md]: sending*

**Comment by the-dagger** [comment]
@utkarshnath just a simple screenshot displaying the received email in user's inbox.
And a short gif which starts the download on clicking the download button would be find I believe.

**Comment by utkarshnath** [comment]
@the-dagger added the required GIF

**Pull request merged**